geopy/geocoders/googlev3.py

```diff
@@ -179,6 +180,11 @@ def geocode(
             .. versionchanged:: 1.14.0
                Now ``query`` is optional if ``components`` param is set.
 
+        :param str place_id: Address information is obtained using google
```

Comment:
The order of params in the docstring should match the order of arguments in the function signature. Could you please fix that? This `:param` block should be the last one.